### ISSUE TITLE
Adding multicursor breaks totally unrelated window buffer virtual text

### ISSUE THREAD

**Issue by dmtrKovalenko** [jake-stewart/multicursor.nvim#64]
- [x] Did you check Readme and existing issues?
    
**Describe the bug**
It took me a while to reproduce this issue but once in my configuration I use this plugin my other window started having issues with virtual text. It includes:

- treesitter context virtual text breadcrumbs 
- lualine configuration for trouble.nvim windows for some reason it shows lualine in the filetype windows it supposed not to show at all and they are freeze to the state of the main window **(it is incredibly annoying)**

here is what it looks like 

https://github.com/user-attachments/assets/ad569b34-ad87-47b9-a719-86302652462e

**To Reproduce**

The easiest way is to run nvim with my config https://github.com/dmtrKovalenko/my-nvim-config cause it only conflicts 

**Expected behavior**
There are no multicursor issues

**Screenshots**
see the video above 

**Nvim version:**
NVIM v0.10.2
Build type: Release
LuaJIT 2.1.1727870382
Run "nvim -V1 -v" for more info

**Multicursor version:**
        branch main
        commit 085fdb7

**Additional context**

**Comment by jake-stewart** [comment]
@dmtrKovalenko should be fixed now, try `:Lazy update`.

The issue was caused because I manually forced the statusline to update. There was an undocumented feature where users could integrate multicursor information in their statuslines but I abandoned this for the signs instead. I just removed the statusline force update.

Let me know if there are any more issues. thanks for the example & reproducible config.
